src/toil_scripts/lib/files.py

```diff
@@ -49,6 +53,49 @@ def copy_file_job(job, name, file_id, output_dir):
     copy_files([fpath], output_dir)
 
 
+def copy_to(filename, output_dir, work_dir=None):
+    """
+    Moves files from the working directory to the output directory.
+
+    :param work_dir: the working directory
+    :param output_dir: the output directory
+    :param filenames: remaining arguments are filenames
+    """
+    if os.path.isabs(filename):
```

Comment:
Actually, since this is a library function, it might be more correct to `assert work_dir is None` in this branch of the conditional.